README.md

```diff
@@ -45,7 +45,9 @@ It correctly bundles React in production mode and optimizes the build for the be
 
 ## Workflow
 
-- **Create new feature-branch from DEVELOP branch**
+- **Create new ```feature-branch``` from ```develop``` branch that describes your work.**
+Example:
+![alt text](https://res.cloudinary.com/elbon/image/upload/v1566291009/branch-naming.png)
 
 - **Adding a New Feature**
   - Create Component
```

Comment:
can you add images or code for all of these steps as well

Reply:
Will do. Thanks